Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/GatewayService.java

```diff
@@ -68,17 +69,20 @@
     private PatientService patientService;
     private DocumentQueryService documentQueryService;
     private DocumentRetrieveService documentRetrieveService;
+
     //Should be moved to a constant file later
-    public static final String CONTENT_TYPE_IMAGE_PNG = "image/png";
-    public static final String CONTENT_TYPE_IMAGE_JPEG = "image/jpeg";
-    public static final String CONTENT_TYPE_IMAGE_GIF = "image/gif";
-    public static final String CONTENT_TYPE_TEXT_XML = "text/xml";
-    public static final String CONTENT_TYPE_TEXT_PLAIN = "text/plain";
-    public static final String CONTENT_TYPE_TEXT_HTML = "text/html";
-    public static final String CONTENT_TYPE_APPLICATION_XML = "application/xml";
-    public static final String CONTENT_TYPE_APPLICATION_XHTML_XML = "application/xhtml+xml";
-    public static final String CONTENT_TYPE_APPLICATION_OCTET_STREAM = "application/octet-stream";
+    public static final String CONTENT_TYPE_IMAGE_PNG = org.springframework.http.MediaType.IMAGE_PNG.toString();
+    public static final String CONTENT_TYPE_IMAGE_JPEG = org.springframework.http.MediaType.IMAGE_JPEG.toString();
+    public static final String CONTENT_TYPE_IMAGE_GIF = org.springframework.http.MediaType.IMAGE_GIF.toString();
+
+    public static final String CONTENT_TYPE_TEXT_XML = MediaType.TEXT_XML;
+    public static final String CONTENT_TYPE_TEXT_PLAIN = MediaType.TEXT_PLAIN;
+    public static final String CONTENT_TYPE_TEXT_HTML = MediaType.TEXT_HTML;
+    public static final String CONTENT_TYPE_APPLICATION_XML = MediaType.APPLICATION_XML;
+    public static final String CONTENT_TYPE_APPLICATION_XHTML_XML = MediaType.APPLICATION_XHTML_XML;
+    public static final String CONTENT_TYPE_APPLICATION_OCTET_STREAM = MediaType.APPLICATION_OCTET_STREAM;
     public static final String CONTENT_TYPE_APPLICATION_PDF = "application/pdf";
```

Comment:
Odd that PDF doesn't have a mime-type constant.  No need to do anything for this PR, but we should consider switching to "com.google.common.net.MediaType" in the future, should we need to touch this code again.